ws-nextjs-app/pages/[service]/live/[id]/ShareButton/styles.ts

```diff
@@ -0,0 +1,42 @@
+import { Theme, css } from '@emotion/react';
+import pixelsToRem from '../../../../../../src/app/utilities/pixelsToRem';
+
+export default ({ palette, fontSizes, fontVariants, spacings, mq }: Theme) =>
+  css({
+    color: palette.BLACK,
+    ...fontSizes.pica,
+    ...fontVariants.sansBold,
+    padding: `${spacings.FULL + spacings.HALF}rem`,
+    marginTop: `${spacings.FULL}rem`,
+    marginBottom: `${spacings.TRIPLE}rem`,
+    marginInlineStart: `${spacings.DOUBLE}rem`,
+    border: `${pixelsToRem(2)}rem solid ${palette.BLACK}`,
+    backgroundColor: 'transparent',
+    cursor: 'pointer',
+    '&:hover, &:focus-visible': {
+      border: `${pixelsToRem(2)}rem solid ${palette.BRAND_BACKGROUND}`,
+      backgroundColor: palette.BRAND_BACKGROUND,
+      color: palette.WHITE,
+      span: {
+        textDecoration: 'underline',
+      },
+      path: {
+        fill: palette.WHITE,
+      },
+    },
+    [mq.GROUP_4_MIN_WIDTH]: {
+      marginBottom: `${spacings.QUADRUPLE}rem`,
+    },
+    span: {
+      verticalAlign: 'middle',
+    },
+    svg: {
```

Comment:
On hover or focus in Firefox when you change colour preferences (haven't checked WHC) you can't see the icon, are we able to fix this?

Reply:
I've fixed this, however there seems to be an issue with Firefox and border colour of buttons, when colour overrides are turned on firefox enforces a grey coloured border, I've also tried using outline, and box shadow to override it, however they don't work:

![image](https://github.com/bbc/simorgh/assets/4706195/74635784-6df0-4d85-a208-ae0416441cce)

Reply:
Are you able to apply styling as per the 'submit' button on uploader, that has an outline that looks ok as far as I can tell?

Reply:
Okay messing around with trial and error, I've managed to get the colour of text to apply to the border. We actually need to fix the border on the submit button, as currently it's set to black and won't adjust to the users colour preference.

Reply:
Cna you raise a ticket for that please when you are done with this @HarveyPeachey

Reply:
Yes will do